src/main/java/com/bartek/Medical/controller/HospitalController.java

```diff
@@ -0,0 +1,32 @@
+package com.bartek.Medical.controller;
+
+import com.bartek.Medical.model.DoctorDTO;
+import com.bartek.Medical.model.Hospital;
+import com.bartek.Medical.model.HospitalDto;
+import com.bartek.Medical.service.HospitalService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/hospitals")
+public class HospitalController {
+    private final HospitalService hospitalService;
+    @PostMapping
+    public ResponseEntity<HospitalDto> addHospital(@RequestBody HospitalDto hospitalDto){
```

Comment:
Dlaczego uzyles tutaj ResponseEntity prosze o przygotowanie krotkiej notatki o tym:
1. czym jest response entity
2. Kiedy uzywamy
3. Dlaczego uzyles w tym przypadku

Reply:
odnosi sie to do wszystkich metod z kontrolera

Reply:
poczytaj o korelacji miedzy ResponseEntity a RestController